martin/src/bin/martin-cp.rs

```diff
@@ -442,6 +441,47 @@ async fn main() {
     }
 }
 
+fn setup_logging() {
```

Comment:
this is a dup - lets not copy paste large functions like this if we can share the code